com.unity.mobile.notifications/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this package will be documented in this file.
 
+## [1.4.4] - 2022-09-30
+
+### Changes & Improvements:
```

Comment:
Changelog changes should come together with actual code changes, not in the separate PR